style.scss

```
@@ -0,0 +1,346 @@
+$cBrown: #bfb2ab;
+
+* {
+    box-sizing: border-box;
+    font-family: 'Oswald', sans-serif;
+    font-family: 'Rasa', serif;
+    font-family: 'Spectral', serif;
+    margin: 0;
+    padding: 0;
+}
+
+
+
+// ========================= mobile ============================================
+@media screen and (max-width:1030px) {
+// ========================= top page ====================
+body {
+    position: relative;
+}
+
+.page1 {
+    height: 100vh;
+
+    .oneArea {
+        height: 100vh;
+        position: absolute;
+        top: 0px;
+        right: 0px;
+        border-bottom: 100vh solid $cBrown;  /* 三角を白にする */ 
+        border-left: 100vw solid transparent;
+    }
+
+    .top {
+        
+        & img {
+            position: absolute;
+            top: 30vh;
+            left: 8px;
+            width: 200px;
+        }
+        &-text {
+
+            & h1 {
+                font-size: 20px;
+                width: 190px;
+                position: absolute;
+                top: 20vh;
+                left: 20px;
+            }
+            & p {
+                font-size: 12px;
+                color: white;
+                width: 180px;
+                position: absolute;
+                top: 65vh;
+                right: 8px;
+            }
+        }
+    }
+} 
+
+// ============================= New collections =====================
+.new {
+
+    margin-top: 50px;
+
+    & h2 {
+        text-align: center;
+        border-bottom: 1px solid $cBrown;
+        width: 200px;
+        margin: 0 auto;
+    }
+
+    & img {
+        border: 1px solid $cBrown;
+        width: 100px;
+        height: 100px;
+        margin-top: 15px;
+    }
+
+    &-collections {
+        display: flex;
+        flex-wrap: wrap;
+        gap: 5.1%;
+        margin-left: 38px;
+        margin-top: 30px;
+
+        & h6 {
+            width: 100px;
+            height: 100px;
+            background-color: $cBrown;
+            margin-top: 15px;
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            color: white;
+            font-size: 10px;
+        }
+    }
+}
+
+// =================== Best collections ================
+.best {
+    margin-top: 50px;
+
+    & h2 {
+        text-align: center;
+        border-bottom: 1px solid $cBrown;
+        width: 200px;
+        margin: 0 auto;
+    }
+
+    &-top {
+
+        & img {
+            display: none;
+        }
+
+        &-text {
+
+            margin: 20px;
```

I want you to add margin more for left and right.
```suggestion
            margin: 20px 40px;
```

---

I changed to margin: 20px 60px; . Please let e to if 60px is too much.